code/io/tree_hdf5.c

```diff
@@ -0,0 +1,513 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <math.h>
+#include <time.h>
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <unistd.h>
+#include <assert.h>
+
+#include "../core_allvars.h"
+#include "../core_proto.h"
+#include "tree_hdf5.h"
+
+// Local Variables //
+hid_t hdf5_file;
```

Comment:
Should this be declared `static` -- which means that this `hdf5_file` variable can only be accessed within this `tree_hdf5.c` file?